plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/GoogleLoginService.java

```diff
@@ -114,7 +114,7 @@ private Credential createCredential(GoogleTokenResponse tokenResponse) {
    * are {@code final}, so Mockito can't mock them.)
    */
   @VisibleForTesting
```

Comment:
Yes, I need it in a test from a different package (`...login.ui`). But it would make sense to extract it to its own class.